go.mod

```diff
@@ -0,0 +1,53 @@
+module github.com/kubernetes-sigs/cri-tools
+
+go 1.12
+
+require (
+	github.com/docker/distribution v0.0.0-20190711223531-1fb7fffdb266
+	github.com/docker/docker v0.7.3-0.20190731001754-589f1dad8dad
+	github.com/docker/go-units v0.4.0
+	github.com/ghodss/yaml v1.0.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/protobuf v1.3.2
+	github.com/onsi/ginkgo v1.8.0
+	github.com/onsi/gomega v1.5.0
+	github.com/opencontainers/go-digest v1.0.0-rc1
+	github.com/opencontainers/selinux v1.2.2
+	github.com/pborman/uuid v1.2.0
+	github.com/sirupsen/logrus v1.4.2
+	github.com/urfave/cli v0.0.0-20180821064027-934abfb2f102
+	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
+	golang.org/x/sys v0.0.0-20190730183949-1393eb018365
+	google.golang.org/grpc v1.22.1
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/cri-api v0.0.0
+	k8s.io/kubernetes v0.0.0
```

Comment:
v0.0.0 means latest. For k8s.io dependencies, do we need to change this to a specific version, e.g. v1.15.0?

Reply:
Not necessarily since we override it in the `replace` section: `k8s.io/kubernetes => k8s.io/kubernetes v1.15.1`